lib/fileset/default.nix

```
@@ -412,6 +417,38 @@ in {
         filter = sourceFilter;
       };
 
+
+  /*
+    The list of file paths contained in the given file set.
+
+    :::{.note}
+    This function evaluates the file set _strictly_.
```

It's not really a property of evaluation, but a property of this function.

```suggestion
    This function evaluates the whole file set.
```

```suggestion
    This function is strict in the entire file set.
```